src/model/CreateOrderRequest.js

```diff
@@ -6,7 +6,7 @@
  */
 
 import ApiClient from '../ApiClient';
-import V1OrdersIssuedTo from './V1OrdersIssuedTo';
+import OrderIssuedTo from './OrderIssuedTo';
```

Comment:
Avoids introducing this weird solo V1 file (this change is due to using a reusable schema for `order_issued_to` in swagger).